src/e2e-test/java/io/cdap/plugin/utils/State.java

```diff
@@ -0,0 +1,419 @@
+/**
+ * Autogenerated by Avro
+ * <p>
+ * DO NOT EDIT DIRECTLY
+ */
+package io.cdap.plugin.utils;
+
+import org.apache.avro.message.BinaryMessageDecoder;
+import org.apache.avro.message.BinaryMessageEncoder;
+import org.apache.avro.message.SchemaStore;
+import org.apache.avro.specific.SpecificData;
+import org.apache.avro.util.Utf8;
+
+/** A list of states in the United States of America. */
+@org.apache.avro.specific.AvroGenerated
+public class State extends org.apache.avro.specific.SpecificRecordBase
+  implements org.apache.avro.specific.SpecificRecord {
+  public static final org.apache.avro.Schema SCHEMA = new org.apache.avro.Schema.Parser()
+    .parse("{\"type\":\"record\",\"name\":\"State\",\"namespace\":\"utilities\",\"doc\":\"A list of states in the" +
+             " United States of America.\",\"fields\":[{\"name\":\"name\",\"type\":\"string\",\"doc\":\"The common " +
```

Comment:
store the schema in the resource file and call it from there.

Reply:
this is the autogenerated file from avro tools.